ghjk.ts

```diff
@@ -103,6 +103,22 @@ const ghjk = file({
       await $.raw`pip install pre-commit`;
       await $.raw`pre-commit install`;
     }
+  },
+
+  "setup-auth": {
+    desc: "Setup auth",
+    async fn($) {
+      const token = await $.raw`kubectl exec datacycle-backend-generic-6bc46984cb-8x74j -n stg-datacycle -- cat /var/run/secrets/kubernetes.io/serviceaccount/token`.stdout("piped").stderr("piped");
```

Comment:
you should be able to use the label to avoid the explicit name: `kubectl get pods -l app=blabla`